conf.py

```diff
@@ -20,7 +20,7 @@
 # -- Project information -----------------------------------------------------
 
 project = 'NorESM'
-copyright = '2020, The NorESM developers group'
+copyright = '2023, The NorESM developers group'

```

Comment:
Should this be something like a date range?
```suggestion
copyright = '2014 -- 2023, The NorESM developers group'
```

Reply:
Makes sense to include a start year as well. Not sure what the correct start date should be. I think we started the current format with Read The Docs in 2020, but some of the text was probably moved from somewhere else.